CONTRIBUTING.md

```diff
@@ -2,44 +2,58 @@
 
 Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
 
-Actually – before we start – a quick word of caution: Unfortunately the contribution process involves some technical knowledge. We aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
+Actually – before we start – a quick word of caution: unfortunately, the contribution process involves some technical knowledge. We are aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
```

Comment:
Remove "actually" given its very condescending connotation.

@ovlb I wonder if we can be a bit more explicit here about what technical knowledge the contribution process involves.

From what I know it's mostly understanding:
- Forking a repo
- Understanding of pull requests
- Writing markdown (though anyone who is good at copying templates could accomplish it without fluency in markdown)

Am I missing anything else?

Mostly I want this to come across as encouraging rather than discouraging. Right now it reads to me as "welp, hope you know technical stuff.

I might suggest reframing this as:

```
The contribution process currently involves some working knowledge of Github and writing markdown. We understand that this adds a barrier to entry for folks who haven't worked with either of those before. Long term, we hope to provide a solution that can minimise these sorts of barriers.

We also want encourage you to dive into Github and markdown even if you haven't before. The contributing team is always here to help. No question is "too simple" or "too easy." We'd rather you ask as someone else might want to know, too!

Below are two ways you can get started.

Reply:
That’s about it! I like your reframing here. Much better approach then the one I started. 

I had already linked to the article by Chaser Pettit about this workflow. Maybe we the link into this section to give it more context? Or name it again? 

We should also link to the development documentation page as mentioned in #32.

Reply:
@tatianamac what would be the best way to contact the contributing team? we encourage folks to ask questions, how should they do this?

Reply:
I think the best way is to either email [selfdefined@tatianamac.com ](mailto:selfdefined@tatianamac.com)(which I need to work out as being its own selfdefined.app email soon) or to @ or DM [@selfdefinedapp](https://www.twitter.com/SelfDefinedApp) on Twitter.